vnext/Microsoft.ReactNative/Views/Image/ReactImage.cpp

```diff
@@ -112,6 +112,19 @@ void ReactImage::TintColor(winrt::Color value) {
   }
 }
 
+void ReactImage::IsPlaying(bool isPlaying) {
+  m_isPlaying = isPlaying;
+  if (const auto brush{Background().try_as<winrt::ImageBrush>()}) {
+    if (const auto bitmapImage{brush.ImageSource().try_as<winrt::BitmapImage>()}) {
```

Comment:
Update `AutoPlay` here as well, so if the image source changes, the image doesn't start playing while `isPlaying` is set to false.